src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -644,3 +644,51 @@
         text: |-
                az cosmosdb mongodb collection merge -g my-resource-group -a my-account -d my-db --name my-mongodb-collection
 """
+
+helps['cosmosdb sql container retrieve-partition-throughput'] = """
+    type: command
+    short-summary: "Retrieves the partition throughput of a sql container."
+    examples:
+      - name: Retrieves container container_name's throughput for specific physical partitions
+        text: |-
+               az cosmosdb sql container retrieve-partition-throughput --account-name account_name --database-name db_name --name container_name  --resource-group rg_name --physical-partition-ids "8,9"
+      - name: Retrieves container container_name's throughput for all physical partitions
+        text: |-
+               az cosmosdb sql container retrieve-partition-throughput --account-name account_name --database-name db_name --name container_name  --resource-group rg_name --all-partitions
+"""
```

Comment:
```suggestion
helps['cosmosdb sql container retrieve-partition-throughput'] = """
    type: command
    short-summary: "Retrieve the partition throughput of a sql container."
    examples:
      - name: Retrieve container container_name's throughput for specific physical partitions
        text: |-
               az cosmosdb sql container retrieve-partition-throughput --account-name account_name --database-name db_name --name container_name  --resource-group rg_name --physical-partition-ids "8,9"
      - name: Retrieve container container_name's throughput for all physical partitions
        text: |-
               az cosmosdb sql container retrieve-partition-throughput --account-name account_name --database-name db_name --name container_name  --resource-group rg_name --all-partitions
"""
```
Don't use third-person singular. Same in other help entries